driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -1393,92 +1385,52 @@ public Builder withNoCompact() {
       return this;
     }
 
-    /**
```

Comment:
Removing of `failIfCloud` and `withCloudSecureConnectBundle` should be probably a part of `
Remove Datastax cloud implementation methods` commit.